### PR TITLE
Add TownePlace Suites (Marriott Brand)

### PR DESCRIPTION
**Pull request by mds08011**
TownePlace Suites is a brand of specialized hotel in owned by Marriott.  I am a bit unsure if the name and brand should be “TownePlace Suites” or “TownePlace Suites by Marriott” but the logo states the latter.

According to the website there are “over 400 locations across the US and Canada”

### CODE REVIEW

File: brands/tourism/hotel.json [hunk 1]
```diff
@@ -520,6 +520,18 @@
       "tourism": "hotel"
     }
   },
+  "tourism/hotel|TownePlace Suites by Marriott": {
```
Comment (tas50) [inline brands/tourism/hotel.json]: I think we want to just make this TownePlace Suites and then have a matchname for TownePlace Suites by Marriot and TownePlace Suites Marriot.

Comment (mds08011) [reply brands/tourism/hotel.json]: I changed the name and added "TownePlace Suites Marriott" and "TownePlace Marriott" as match names.  "TownePlace Suites by Marriott" will be matched via `official_name`

File: brands/tourism/hotel.json [hunk 2]
```diff
@@ -520,6 +520,21 @@
       "tourism": "hotel"
     }
   },
+  "tourism/hotel|TownePlace Suites": {
+    "countryCodes": ["ca", "us"],
+    "matchNames": [
+      "TownePlace Suites Marriott"
```
Comment (tas50) [inline brands/tourism/hotel.json]: There should be a comma at the end of this line. It's causing travis failures

Comment (mds08011) [reply brands/tourism/hotel.json]: Thanks!  Just added it

**Comment by tas50** [comment]
Thanks @mds08011

**Pull request merged**